bin/sozo/src/commands/account.rs

```diff
@@ -83,12 +86,22 @@ pub enum AccountCommand {
 
 impl AccountArgs {
     pub fn run(self, config: &Config) -> Result<()> {
+        trace!(target: LOG_TARGET, "AccountArgs command: {:?}", self.command);
         let env_metadata = utils::load_metadata_from_config(config)?;
+        trace!(target: LOG_TARGET, "Fetched environment metadata");
 
         config.tokio_handle().block_on(async {
             match self.command {
                 AccountCommand::New { signer, force, file } => {
+                    trace!(
+                        target: LOG_TARGET,
+                        "Executing New command with signer: {:?}, force: {}, file: {:?}",
+                        signer,
+                        force,
+                        file
+                    );
```

Comment:
Let's use structured logging here and elsewhere

```suggestion
                    trace!(
                        target: LOG_TARGET,
                        "Executing New command",
                        signer=?signer,
                        force=%force,
                        file=?file
                    );
```